ui/app/AppLayouts/Chat/CommunityComponents/CreateCommunityPopup.qml

```diff
@@ -65,9 +66,8 @@ StatusModal {
 
         contentHeight: content.height
         bottomPadding: 8
-        height: Math.min(content.height, 432)
```

Comment:
On add/edit community popup load, some components below the thumbnail image picker were not appearing until the app window was resized. Removing the height of the ScrollView fixed the issue (on mac).

Reply:
Removing the height is an issue though because then it'll never be scrollable and always grow as big as its content is

Reply:
That's not what I've observed, it does remain scrollable, but I've only tested on macos. The [documentation](https://doc.qt.io/qt-5/qml-qtquick-controls2-scrollview.html#sizing) mentions that the `contentHeight` property needs to be set, but does not mention that `height` needs to be set:
> If only a single item is used within a ScrollView, the content size is automatically calculated based on the implicit size of its contained item. However, if more than one item is used (or an implicit size is not provided), the contentWidth and contentHeight properties must be set to the combined size of its contained items.

Reply:
Oh maybe it works then because there's only one item (`Column`) in there?
Cause otherwise, how would the ScrollView know how high it should be (unless there's `anchors` provided).

Anyways, if you've tested it and it works, then that's great! Simplifies things.